add utility to upload files by policies (#14)
XcooBee/xcoobee-js-sdk: README.md

#### response
public PGP or empty string

## Static utilities and helpers

### uploadFiles(files, policies)

Allows user to upload files by policies received from XcooBee API without authorization.

Node JS example:
```

> Xcoobee.Utilities.uploadFiles(['test.png'], [{...}]).then(...); // NOTE, that files should be in folder where you run script.

Reviewer: ```suggestion
Xcoobee.Utilities.uploadFiles(['test.png'], [{...}]).then(...); // NOTE: Relative file paths are relative to directory where you run script.
```

```
Browser example:
```

> Xcoobee.sdk.Utilities.uploadFiles([File], [{...}]).then(...); // NOTE, file should be instance of Web File.

Reviewer: ```suggestion
Xcoobee.sdk.Utilities.uploadFiles([file], [{...}]).then(...); // NOTE: file should be instance of Web File.
```
